### DISCUSSION TITLE
zen мир

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2_classic#116] [General]
не нашёл топика

**Comment by Kor-Angar** [comment]
Дырки возле башни Декстера

**Reply by Kor-Angar** [reply]
![20240102030847_1](https://github.com/dosinabox/g2_classic/assets/49514764/75722d1b-080b-4f93-a19f-6bf9d463e73b)

**Reply by Kor-Angar** [reply]
![20240102030812_1](https://github.com/dosinabox/g2_classic/assets/49514764/f476e74e-7fe5-494b-b4c6-2a4860a04fdf)

**Reply by dosinabox** [reply]
Исправлено в обновлении г2нв. В идеале нужно переносить миры оттуда и делать точечные изменения для соответствия классике. Заново отдельно миры никто не будет править.

Знал про эту пещеру под мостом к лагерю Декстера? Я только сейчас обнаружила случайно :)
![Screenshot_3](https://github.com/dosinabox/g2_classic/assets/39790055/e70175f9-4e0f-4653-a144-4bdd1d2542e5)

**Reply by Kor-Angar** [reply]
> Знал про эту пещеру под мостом к лагерю Декстера? Я только сейчас обнаружила случайно :)

нет.

**Comment by Kor-Angar** [comment]
если будешь править мир миненталя добавь плиз паре сундуков уникальные имена

**Reply by Kor-Angar** [reply]
в форте
![20241122221544_1](https://github.com/user-attachments/assets/6f344d40-6854-48f5-a1c2-2475fe4e7ab7)

**Reply by Kor-Angar** [reply]
и в пещере там где амулет Ангара

**Reply by dosinabox** [reply]
> в форте

А как эти назвать?

**Reply by Kor-Angar** [reply]
демон вставляется сюда

	Wld_InsertNpc(Demon,"LOCATION_19_03_ROOM6_BARRELCHAMBER2");

**Reply by Kor-Angar** [reply]
демон вставляется сюда 	Wld_InsertNpc(Demon,"LOCATION_19_03_ROOM6_BARRELCHAMBER2");

назвать можно так например `CHESTBIG__ROOM6_BARRELCHAMBER2`
